src/Modules/Scheduler.asm

```diff
@@ -196,39 +227,52 @@ scheduler_steps_odd_status_frame_max_loaded:
     mov  Ext_Telemetry_H, #0Eh          ; Set telemetry high value to status frame ID
 
     ; Now restart ADC conversion
-    sjmp scheduler_steps_odd_restart_ADC
+    Stop_Adc
```

Comment:
How about a restart macro for the ADC since it is used a couple times from here on?

Reply:
Done